test/index.spec.tsx

```diff
@@ -192,7 +213,9 @@ describe('DOM Events', () => {
           expect(onFocus).toBeTruthy();
           done();
         }}
-      />);
+      />,
+      { attachTo: container }
```

Comment:
Focus-related Enzyme tests don't work without being attached to an actual DOM element in the document.